source/contributors/_index.md

```diff
@@ -50,11 +50,30 @@ reasonable first step towards contributing. Keep notes of that first
 user experience, as that is valuable information for the project, and
 might be a good place to start your contributions.
 
-### The Code
+### What can I contribute?
 
-Most (not all) Apache projects have their code on
-[GitHub](https://github.com/apache). Search for the project name in the
-**apache** GitHub organization.
+Most people's first thought when contributing to open source projects is
+code. And, indeed, that's where most people get involved. Most (not all) 
+Apache projects have their code on [GitHub](https://github.com/apache).
+Search for the project name in the **apache** GitHub organization.
+
+There are, however, many other areas where you can contribute to a
+project. These include, but are not limited to:
+
+* Documentation
+* Marketing and social media promotion
+* Event planning
+* Technical support on mailing lists and other forums
+* Vendor/Sponsor relations
+* Project management (organizing the next sprint or feature set, keeping
+  track of who is working on what, updating the roadmap)
+* Community management
+
+Projects should identify, recruit for, and celebrate contributions in
+all of these areas, and reward non-code contributors with the same
+rights and privileges that they do for code contributors. 
+
+**Being a committer is *not* just for code contributors!**
```

Comment:
We sometimes say that it's more about "being committed to the project" than "committing code", might be good to add this here?